src/portable/options.rs

```diff
@@ -431,6 +434,31 @@ pub struct Info {
     pub version: Option<ver::Filter>,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub enum ShowCredentialsType {
+    JSON,
+    InsecureDSN,
+}
+
+#[derive(EdbClap, Clone, Debug)]
+pub struct ShowCredentials {
+    #[clap(long, possible_values=&[
+    "json", "insecure_dsn",
+    ])]
+    pub _type: Option<ShowCredentialsType>
+}
+
+impl FromStr for ShowCredentialsType {
+    type Err = anyhow::Error;
+    fn from_str(v: &str) -> anyhow::Result<ShowCredentialsType> {
+        use ShowCredentialsType::*;
+        match v {
+            "json" => Ok(JSON),
+            "insecure_dsn" => Ok(InsecureDSN),
```

Comment:
```suggestion
            "insecure-dsn" => Ok(InsecureDSN),
```
I think we use kebab-case everywhere else